objective.py

```diff
@@ -111,7 +107,7 @@ def compute(self, model):
                 ["train", "test"], [self.tf_dataset, self.tf_test_dataset]
             ):
                 metrics = model.evaluate(
-                    self.tf_dataset.batch(self.batch_size),
+                    dataset.batch(100),
```

Comment:
wouldn't we want to have this as a settable attribute in the init?

Reply:
This also would allow to share between the tf and torch computation

Reply:
This is simply for test loss computation so I would not worry too much.
But you are right that we should share this between tf and torch.